MachineLearningPrototype/utils/preprocess_data.py

```diff
@@ -8,3 +8,9 @@ def remove_time_stamp(content):
     new_content = re.sub(r"\d{2}:\d{2}:\d{2}", "", content)
     new_content = re.sub(r"\[\d{4}-\d{2}-\d{2}.*?\]", "", new_content)
     return new_content
+
+def extract_quotation_content(content):
+    pattern = "\`\`\`([\s\S]*?)\`\`\`"
+    substring = re.findall(pattern, content)
+    result=' '.join(substring)
+    return result
```

Comment:
- Condition check may be needed, e..g `if (len(substrings) < 2)`, return substrings, else join substrings and return. 
- Could you also address Lan's comment of printing exception info? Thanks.